client/src/main/java/com/alibaba/nacos/client/naming/net/HttpClient.java

```diff
@@ -104,7 +109,9 @@ public static HttpResult request(String url, List<String> headers, Map<String, S
 
             return new HttpResult(500, e.toString(), Collections.<String, String>emptyMap());
         } finally {
-            IoUtils.closeQuietly(conn);
+            if (conn != null) {
```

Comment:
Please user IoUtils.closeQuietly to keep connection reused. Refer to https://stackoverflow.com/questions/4767553/safe-use-of-httpurlconnection